src/main/java/uk/ac/ebi/eva/pipeline/io/writers/VepAnnotationFileWriter.java

```diff
@@ -0,0 +1,225 @@
+
+import java.io.BufferedInputStream;
+import java.io.BufferedOutputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.zip.GZIPOutputStream;
+
+public class VepAnnotationFileWriter implements ItemStreamWriter<VariantWrapper> {
+    private static final Logger logger = LoggerFactory.getLogger(VepAnnotationFileWriter.class);
+
+    private AnnotationParameters annotationParameters;
+
+    private Integer chunkSize;
+
+    private Process process;
+
+    private OutputStream perlStdin;
+
+    private final Long timeoutInSeconds;
+
+    private Thread outputCapturer;
+
+    private AtomicBoolean writingOk;
+
+    private boolean alreadyClosed;
+
+    public VepAnnotationFileWriter(Long timeoutInSeconds, Integer chunkSize, AnnotationParameters annotationParameters) {
+        if (timeoutInSeconds <= 0) {
+            throw new IllegalArgumentException(
+                    "timeout (" + timeoutInSeconds + " seconds) must be strictly greater than 0");
+        }
+        this.timeoutInSeconds = timeoutInSeconds;
+        this.chunkSize = chunkSize;
+        this.annotationParameters = annotationParameters;
+        alreadyClosed = false;
+    }
+
+    @Override
+    public void open(ExecutionContext executionContext) throws ItemStreamException {
+        ProcessBuilder processBuilder = new ProcessBuilder("perl", annotationParameters.getVepPath(),
+                "--cache",
+                "--cache_version", annotationParameters.getVepCacheVersion(),
+                "-dir", annotationParameters.getVepCachePath(),
+                "--species", annotationParameters.getVepCacheSpecies(),
+                "--fasta", annotationParameters.getInputFasta(),
+                "--fork", annotationParameters.getVepNumForks(),
+                "--buffer_size", chunkSize.toString(),
+                "-o", "STDOUT",
+                "--force_overwrite",
+                "--offline",
+                "--everything"
+        );
+
+        logger.debug("VEP annotation parameters = " + Arrays.toString(processBuilder.command().toArray()));
+
+        logger.info("Starting VEP process");
+        process = null;
+        try {
+            process = processBuilder.start();
+        } catch (IOException e) {
+            throw new ItemStreamException(e);
+        }
+
+        perlStdin = new BufferedOutputStream(process.getOutputStream());
+        InputStream perlStdout = process.getInputStream();
+        String vepOutput = annotationParameters.getVepOutput();
+
+        writingOk = new AtomicBoolean(false);
+        outputCapturer = new Thread(() -> {
+            long written = 0;
+            try (GZIPOutputStream outputStream = new GZIPOutputStream(new FileOutputStream(vepOutput))) {
```

Comment:
Change this call to the Constructor with the append parameter so that we don't overwrite the file over and over and over (as we found)